Quantum-Control-Applications-QuAM/Superconducting/pyproject.toml

```diff
@@ -13,17 +12,21 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8"
-quam = { git = "https://github.com/qua-platform/quam.git" }
+python = ">=3.9,<3.13"
+quam = "*"
 qualang-tools = ">=0.19.0"
-xarray = "*"
-lmfit = "*"
-qiskit-experiments = "*"
-tqdm = "*"
-qualibrate = "*"
-qm-qua = "1.2.1"
-colorama = "*"
+qualibrate = ">=0.1.6"
+qm-qua = ">=1.2.1"
 tomli = { version = "^2.0.1", python = "<3.11" }
+quam_libs = { path = "quam_libs", develop=true }
+quam_config = { path = "quam_config", develop=true }
+quam_experiments = { path = "quam_experiments", develop=true }
```

Comment:
Why do we have quam_config and quam_experiments as dependencies rather than modules?
My original thinking was that they don't need separate pyproject.toml files, but can instead be written as two source directories

Reply:
Let's sort the `pyproject.toml` discussion outside of this PR so that we can move on